### PR TITLE
fix(account-sidebar): styling & presentation

### PR DESCRIPTION
**Pull request by frenautvh**
Signed-off-by: frenauvh <florian.renaut@corp.ovh.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | feat/manager-hub
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | internal MANAGER-4794, MANAGER-4795
| License          | BSD 3-Clause

## Description

fix some incorrect style

### CODE REVIEW

**Comment by antleblanc** [comment]
@frenautvh There are conflicts!

File: packages/manager/modules/account-sidebar/src/user-infos/template.html [hunk 1]
```diff
@@ -70,13 +60,13 @@
...
             <span
                 class="manager-hub-user-infos_role oui-status oui-status_warning"
-                data-ng-if="!$ctrl.isExpanded && $ctrl.role"
+                data-ng-if="$ctrl.role"
                 data-ng-bind=":: ('hub_user_role_' + $ctrl.role) | translate"
             >
             </span>
         </div>
     </div>
-    <div data-ng-if="!$ctrl.isExpanded">
```
Comment (antleblanc) [inline packages/manager/modules/account-sidebar/src/user-infos/template.html]: Maybe we can get rid of this `<div>`.

Comment (frenautvh) [reply packages/manager/modules/account-sidebar/src/user-infos/template.html]: yep ty

File: packages/manager/modules/account-sidebar/src/payment-mean/template.html [hunk 2]
```diff
@@ -20,11 +20,11 @@
         />
         <oui-skeleton size="s" data-ng-if="$ctrl.isLoading"> </oui-skeleton>
         <div class="m-auto p-1 minw-0" data-ng-if="!$ctrl.isLoading">
-            <strong
-                class="d-block"
-                data-translate="hub_payment_mean_title"
-            ></strong>
-            <div data-ng-if="$ctrl.paymentMethod">
+            <h3 class="d-block" data-translate="hub_payment_mean_title"></h3>
```
Comment (antleblanc) [inline packages/manager/modules/account-sidebar/src/payment-mean/template.html]:
```suggestion
            <h3 data-translate="hub_payment_mean_title"></h3>
```

**Pull request merged**